### DISCUSSION TITLE
How to Track Elevation?

### DISCUSSION THREAD

**Discussion by nader4** [zepp-health/zeppos-forum#16] [Developer Feedback]
I'd like to track elevation during a workout.  I figured this would be built in seeing as there are workouts for stairs, but this is unfortunately not the case.  As I understand it, my Amazfit Band 7 has a 3 axis accelerometer sensor.  This is used to calculate distance travelled I imagine, but i'd like to try and configure it to calculate the elevation as well.    I'm wondering if I can look at the source code for distance and configure it to track elevation as well.  Is this possible?  I would also like to leverage that same concept for rep counting for weightlifting.  The idea would be to track back and forth motions or up and down motions with the accelerometer.  Is this also possible?  I don't have any experience coding for an accelerometer so that's why i'm asking.  I understand that usually for elevation you need more sensors like barometer or altimeter or something, but just for workouts, i'm wondering if 3 axis accelerometer is good enough if you want to run up and down the stairs.  Please let me know if anyone has any information or documentation on this.  Thank you!

**Comment by MyoungXUE** [comment]
First of all, thank you for your support of the Developer Forum

1. For the time being we have no source code no open plan, so for the developer community there is no way to see the source code
2. Your idea is very cool, the sensor you mentioned like altimeter, accelerometer, we are considering to open it with a proper API and expect to meet with developers next year, but it will not be updated to Amazfit Band 7
